src/manifest.json

```
@@ -3,7 +3,7 @@
   "name": "GOV.UK Toolkit",
   "description": "Switch between GOV.UK environments and content",
   "homepage_url": "https://github.com/alphagov/govuk-browser-extension",
-  "version": "1.12.0",
+  "version": "1.13.0",
```

```suggestion
  "version": "1.12.0",
```

The version needs to be updated in a separate commit: https://github.com/alphagov/govuk-browser-extension#releasing-the-extension